### PR TITLE
🌐 Add French definition for "unreal"

### PR DESCRIPTION
**Pull request by CharlesMangwa**
Hello!

This PR aims to add the French definition for **"unreal"**. 

A link has been added to the existing English definition (and vice versa).

Thanks in advance to the editor! 🌼

### CODE REVIEW

File: 11ty/definitions/fr_FR/ireel.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: iréel
```
Comment (deammer) [inline 11ty/definitions/fr_FR/ireel.md]: Ce mot contient deux "r": https://www.le-dictionnaire.com/definition/irr%C3%A9el

**Review by deammer — CHANGES_REQUESTED** [review]
The spelling needs to be fixed, but looks great otherwise

**Comment by CharlesMangwa** [comment]
Just pushed a correction for the typo caught by @deammer (thanks!)

**Review by deammer — APPROVED** [review]
Thanks for the update! Looks great :shipit:

File: 11ty/definitions/fr_FR/irreel.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Irréel
+slug: irreel
+defined: true
+excerpt: très impressionnant ; étonnant ; plus imposant ou plus fantastique que ce qui est typique de la vie réelle.
+speech: adj
+---
```
Comment (tatianamac) [inline 11ty/definitions/fr_FR/irreel.md:10]: Add definition reference cited by @deammer

**Review by tatianamac — APPROVED** [review]

File: 11ty/definitions/fr_FR/irreel.md (hunk 2 above)
Comment (CharlesMangwa) [reply 11ty/definitions/fr_FR/irreel.md:10]: Reference added and the title is also in lowercase now.

**Pull request merged**